aws/src/main/java/org/apache/iceberg/aws/s3/signer/S3V4RestSignerClient.java

```diff
@@ -113,6 +113,20 @@ boolean keepTokenRefreshed() {
         OAuth2Properties.TOKEN_REFRESH_ENABLED_DEFAULT);
   }
 
+  @Nullable
```

Comment:
I thought `@Nullable` is only needed for public methods, do we also need for private methods?